packages/semantic-ui/src/components/PlayButton.js

```diff
@@ -5,8 +5,17 @@ import { Button } from 'semantic-ui-react';
 import './PlayButton.css';
 
 type Props = {
+  /**
+   * What to do when the button is clicked.
+   */
   onClick?: () => void,
-  size?: string,
+    /**
+   * Size of the button.
+   */
+  size?: 'mini' | 'tiny' | 'small' | 'medium' | 'large' | 'big' | 'huge' | 'massive',
+    /**
+   * Custom styling object.
+   */
   style?: any
 };
 
```

Comment:
Let's add a component comment too to serve as a summary of what the component does.